src/SmartComponents/SystemAdvisor/SystemAdvisor.js

```diff
@@ -176,16 +179,28 @@ const BaseSystemAdvisor = () => {
     kbaLoading = false,
     isFirstLoad = false
   ) => {
-    const builtRows = activeReports.flatMap((value, key) => {
+    let isRecParamPresent =
+      window.location.href.indexOf('recommendations') > -1 ? true : false;
```

Comment:
I kinda feel like this wether or not it is expanded should be solved differently without having to inspect the path.

Reply:
Did you have something else in mind? At least right now I cant think of an alternative that will work on both envs. If an alternative is created we could also probably change how breadcrumbs work. Why dont you like inspecting the path though? Because of how routes are set up right now, I cant think of a scenario where this wouldnt work properly